forge-game/src/main/java/forge/game/trigger/TriggerFullyUnlock.java

```diff
@@ -0,0 +1,41 @@
+package forge.game.trigger;
+
+import java.util.Map;
+
+import forge.game.ability.AbilityKey;
+import forge.game.card.Card;
+import forge.game.spellability.SpellAbility;
+import forge.util.Localizer;
+
+public class TriggerFullyUnlock extends Trigger {
```

Comment:
imo it should be just TriggerUnlock so it can be used for "When you unlock this door..." too
like a Side$ param with Left/Right/Both

Reply:
Imo I think extra Trigger would make more sense

Because for example when you later unlock the second door, it triggers both:
* Unlock (this) Door "Left/Right"
* Fully Unlock Room

Reply:
Hmn, I'm not sure I see the advantage there but it's no big deal anyway :)